### ISSUE TITLE
Create new harvesting rake tasks to use new harvest broker/request classes for WoS

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/sul_pub#195]
- [x] create new rake tasks using new WoS harvest
- [ ] update the wiki
- [x] update the harvesting schedule via whenever

**Comment by darrenleeweber** [comment]
@peetucket - it's not entirely clear how many of the SW rake tasks to implement? Before trying to implement all of them (or as many as possible), it would good to review the rake tasks in the SW namespace and update this issue
- https://github.com/sul-dlss/sul_pub/blob/master/lib/tasks/sw.rake

**Comment by darrenleeweber** [comment]
In slack commentary, we decided to focus only on the harvest tasks (or anything required by the harvest scheduler).

**Comment by peetucket** [comment]
This is the other rake task I have used periodically (every one to two months): wos_profile_id_report

It is a manual load of publications by wos_id to a given cap_profile_id

**Comment by darrenleeweber** [comment]
Let's create a new issue for implementing that additional rake task, so work on closing this issue can focus on the harvest for all authors.

**Comment by darrenleeweber** [comment]
I think updating the wiki should be split out into another issue, to be part of the process for a final sign-off on the WOS work, because a lot of work needs to be done before the new harvest rake task and schedule will work properly.  That is, it's premature to document something in the wiki (and possibly replace old content), without the integration testing to check that it all works.
